src/routes/dashboard/components/Dashboard.js

```diff
@@ -99,6 +102,19 @@ class Dashboard extends React.Component<PropsType> {
       }))
     }
 
+    tiles.push(new TileModel({
```

Comment:
This should be disabled before merging this

Reply:
There is a flag in the build config that should be used.